orbit/pkg/table/pwd_policy/pwd_policy.go

```diff
@@ -42,31 +44,36 @@ func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[strin
 
 	out, err := cmd.Output()
 	if err != nil {
-		return nil, fmt.Errorf("generate failed: %w", err)
+		log.Debug().Err(err).Msg("Running pwpolicy falied")
+		return nil, fmt.Errorf("Running pwpolicy falied: %w", err)
```

Comment:
`return nil, fmt.Errorf("running pwpolicy failed: %w", err)`

(Errors strings should start in lowercase according to some golang linters.)